Automattic/jetpack
# Revue block: Part 1 of retiring the block - adding a placeholder message.

Fixes #27806

#### Changes proposed in this Pull Request:

* This is the first part of retiring the Revue block. This involves outputting a message within the block in the editor, and on the front-end for admins only, explaining that Revue is shutting down.


#### Other information:


#### Jetpack product discussion

Slack conversation: p1671036856312609-slack-C01U2KGS2PQ

Some further information about Revue shutting down here: p58i-dGf-p2 

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* To test, you will need to add a Revue block before applying this PR (as the PR also removes the ability to find the Revue block in the block inserter tool). It is ok if you don't have a Revue account - you should still see a block in the editor with a form to add you Revue username, and the front-end should not show anything in that case.
* Apply this PR. The editor should now show a message and link related to Revue shutting down, suggesting the block should be removed and adding a CTA to migrate subscribers via WordPress.com. On the front-end, if you are logged in as an admin you should see an info notice with similar messaging. View the same post / page whilst not logged in or not as an admin and you shouldn't see that notice.
